### PR TITLE
fix(Actions): make create-app test work every time

### PR DESCRIPTION
**Pull request by SanderElias**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/scullyio/scully/blob/main/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [x] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: .github/workflows/workflow.yml [hunk 1]
```diff
@@ -3,9 +3,6 @@ name: Scully build, test, package, and deploy
 on:
   # Trigger the workflow on push or pull request,
   # but only for the main branch
-  push:
-    branches:
-      - main
```
Comment (Jefiozie) [inline .github/workflows/workflow.yml:8]: Should this not be added again?

File: .github/test-build/action.yml [hunk 2]
```diff
@@ -29,26 +29,53 @@ runs:
...
+        cd ../demo
         echo "install deps"
         npm i rxjs@7  --legacy-peer-deps --force --quiet --silent
         echo "$scullyFolder"
-        cp $scullyFolder/*.tgz .
-        for f in *.tgz; do
-          echo "install package $f"
-          npm install $f --force --legacy-peer-deps --quiet --silent
+        cp ../scully/*.tgz .
+        for f in  scullyio-scully-2*.tgz; do
```
Comment (Jefiozie) [inline .github/test-build/action.yml:51]: Should the `2` be removed?

**Comment by gn-sky** [comment]
it's interesting that `package-lock.json` is pushed without `package.json`. Was source `package.json` merged in prior PR?

File: .github/workflows/workflow.yml (hunk 1 above)
Comment (gn-sky) [reply .github/workflows/workflow.yml:8]: if removal is intentional, I think it's best to adjust comment line above as well

Comment (SanderElias) [reply .github/workflows/workflow.yml:8]: The change is intentional. I agree about the comment, I'll update it.
A push to main is _only_ happening after a merge. A pr will only be merged after all tests are passed.
So, the previous setup would run all the tests again _after_ the merge.
Also, it started the tests after the version bump did its work, and I don't think we need/want to test version bumps.

File: .github/test-build/action.yml (hunk 2 above)
Comment (SanderElias) [reply .github/test-build/action.yml:51]: No, it prevents installing any off the other packages _before_ scully itself

**Review by gn-sky — APPROVED** [review]

**Pull request merged**